build.gradle

```diff
@@ -36,11 +36,11 @@ buildscript {
 }
 
 ext {
-    groupId = 'de.cyface'
-    version = "5.0.0-beta1"
+    // This libraries version
+    cyfaceBackendVersion = "5.0.0-beta1"
 
     // Cyface dependencies
-    utilsVersion = "1.1.3"
```

Comment:
Wenn man das Backend (und unsere anderen library module) als Submodule in der android-app einbindet muss man ja diese Variablen dort definieren (wieder wegen rootProject => nun android-app/build.gradle).

Bisher war es immer etwas komisch, wenn in der app dann mit "version" die Version des Backends gemeint ist. Daher habe ich die Namensgebung eindeutiger gemacht. Die anderen Android Projekte sind bereits entsprechend angepasst, damit die kompatibel sind.